File: Csharp/Stringy strings.cs
// https://www.codewars.com/kata/563b74ddd19a3ad462000054/train/csharp

using System.Text;

public class Kata
{
  public static string Stringy(int size)
  {
      var result = new StringBuilder();
      for (var i = 1; i <= size; i++)
      {
          result.Append(i%2);
      }
      return result.ToString();
  }
}

File: javascript/Stringy strings.js
// https://www.codewars.com/kata/563b74ddd19a3ad462000054/train/javascript

function stringy(size) {
    let res = ''
    for(let i = 0; i < size; i++){
      res += i%2 === 0 ? "1" : "0"
    }
    return res
    }